### PR TITLE
HOM-17: Makes arrow for top level external link white

### PR DESCRIPTION
**Pull request by shc314**
# READY FOR REVIEW

# Summary
- Changes the style of the mega menu top level external link so that the colors match the other panel links. The arrow icon is white and becomes red when the link is hovered over.

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to global-components/masthead/reunion-homecoming-global-header2
3. Verify that the arrow icon to the right of the Register link is white, is no longer lower than the other arrows, and that it is the same red as the other top level menu items in the hover and focus states.

# Associated Issues and/or People
- HOM-17
- @willyumfrancis

### CODE REVIEW

**Comment by willyumfrancis** [comment]
This is a little bug fix where the arrow in our menu is red when it should be white. Attached screenshot.

https://alumni.stanford.edu/editor/?access_key=P8rUgpOF4AoByilQSscWwgtt&path=global-components/masthead/reunion-homecoming-global-header2&_storyblok=516730860&_storyblok_c=globalHeader&_storyblok_version=&_storyblok_lang=default&_storyblok_release=0&_storyblok_rl=1723674642084&_storyblok_tk%5Bspace_id%5D=103430&_storyblok_tk%5Btimestamp%5D=1723673064&_storyblok_tk%5Btoken%5D=970661c8ac91b880f0af4aefa8bb940da9471ce5

Screenshot 2024-09-23 at 13 16 23

**Comment by sherakama** [comment]
Reposting for @willyumfrancis 

A friendly reminder to avoid posting preview URL links in pubic places because they contain a preview key which allows others to see unpublished content.

> This is a little bug fix where the arrow in our menu is red when it should be white. Attached screenshot.
> 
> Story: /global-components/masthead/reunion-homecoming-global-header2
> 
> Screenshot 2024-09-23 at 13 16 23

File: src/components/navigation/MegaMenu/megaMenuLink.styles.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 export const menuLink =
   'su-inline-block su-leading-display su-text-black hocus:su-text-digital-red su-no-underline hocus:su-underline su-font-normal su-text-19 su-group su-backface-hidden';
 export const externalIcon = 'su-text-digital-red su-text-21';
+export const externalIconWhite =
```
Comment (sherakama) [inline src/components/navigation/MegaMenu/megaMenuLink.styles.js:4]: I _think_ you can combine focus and hover here into `group-hocus`...

File: src/components/navigation/MegaMenu/megaMenuLink.styles.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+  'su-text-white su-text-21 su-mb-2 group-hover:su-text-digital-red-xlight group-focus:su-text-digital-red-xlight';
```
Comment (sherakama) [inline src/components/navigation/MegaMenu/megaMenuLink.styles.js:5]:
```suggestion
  'su-text-white su-text-21 su-mb-2 group-hocus:su-text-digital-red-xlight';
```

File: src/components/navigation/MegaMenu/megaMenuLink.styles.js (hunk 1 above)
Comment (sherakama) [reply src/components/navigation/MegaMenu/megaMenuLink.styles.js:4]: Yeah, should be able to https://github.com/SU-SWS/decanter/blob/main/tailwind.config.js#L48

**Review by sherakama — APPROVED** [review]
@shc314 switch over to `group-hocus` and this is GTG.

**Pull request merged**